139-pipeline-identity-tokens/proposal.md

```diff
@@ -0,0 +1,133 @@
+* RFC PR: [concourse/rfcs#139](https://github.com/concourse/rfcs/pull/139)
+
+# Summary
+
+Pipelines should receive signed JWTs ([RFC7519](https://datatracker.ietf.org/doc/html/rfc7519)) from Concourse that contain information about them (team, pipeline-name etc.).
+They could then send this JWTs to outside services to authenticate using their identity as "Concourse-Pipeline X"
+
+
+# Motivation
+Often pipelines have to interact with outside services to do stuff. For example deploy something to AWS.
+As of now you would need to create static credentials for these outside services and place them into concourse's secrets-management (for example inside vault).
+
+However having static (long lived) credentials for something that is critical (like a prod account on AWS) is not state of the art for authentication.
+It would be much better if code running in a pipeline could somehow prove it's identity to the outside service. The outside service could then be configured to grant permissions to a specific pipeline.
+
+Lot's if other services already implement something like this. One well knwon example of this are [Kubernetes's Service Accounts](https://kubernetes.io/docs/concepts/security/service-accounts/#authenticating-credentials). Kubernetes mounts a signed JWT into the pod and the pod can then use this token to authenticate with Kubernetes itself or with any other service that has a trust-relationship.
```

Comment:
typos: 
1. `Lot's if other services` ---> `Lots of other services`
2. `One well knwon example` ---> `One well known example`